pkg/print/table.go

```diff
@@ -41,20 +42,43 @@ func (t Table) apiKeys(apiKeys []api.APIKey) {
 func (t Table) tasks(tasks []api.Task) {
 	tw := tablewriter.NewWriter(os.Stdout)
 	tw.SetBorder(false)
-	tw.SetHeader([]string{"name", "slug", "builder", "arguments"})
+	tw.SetHeader([]string{"name", "slug", "builder", "parameters"})
 
 	for _, t := range tasks {
 		var builder = t.Builder
-
 		if builder == "" {
 			builder = "manual"
 		}
 
+		var parametersStr string
+		if len(t.Parameters) > 0 {
+			parametersTableString := &strings.Builder{}
+			ptw := tablewriter.NewWriter(parametersTableString)
+			ptw.SetBorder(false)
+			ptw.SetHeader([]string{"name", "slug", "type", "required", "default"})
+			for _, p := range t.Parameters {
+				var defaultStr = fmt.Sprintf("%v", p.Default)
```

Comment:
NB this won't always work when Default is something more exotic like an upload ID, and it's OK but not great for dates, etc. Filed AIR-760 to track for later, `%v` seems OK for now.

And also, minor nit:

```suggestion
				defaultStr := fmt.Sprintf("%v", p.Default)
```

(@colinking would be nice to have lint enforce consistency on short decl over vars :) )